capice.py

```diff
@@ -41,39 +47,39 @@ def main():
     verbose = cla.get_argument('verbose')
     force = cla.get_argument('force')
     train = cla.get_argument('train')
+    config_argument = convert_cla_to_str(cla.get_argument('config'))
+
+    # Initializing manager
+
+    manager = CapiceManager()
+    manager.config_loc = config_argument
+    manager.verbose = verbose
+    manager.force = force
```

Comment:
More of a design-technical point. Current flow before running main/train is:
1. Parse command line arguments
2. Retrieve & convert each argument individually (through using a separate utilities class)
3. Initializing the global manager (CapiceManager) & override some settings
4. Parse the config file
5. Retrieve & convert all items from the config file individually (through using separate checker class)
6. Override remaining settings of manager

Might it not be cleaner to do the following?
1. Run command line parser
2. Run config file parser
3. Initialize CapiceManager that requires these objects during it's initialization (or make the config file optional) and ensures everything is set accordingly & validates whether all required fields are set (or offers a method to call this externally that returns `true`/`false`). You could even remove the setters if preferred to indicate variables shouldn't be adjusted after it was initialized or replace the default setters with ones that only accept the command line/config file parser.